spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/RewriteDataFilesProcedure.java

```diff
@@ -130,9 +131,10 @@ private RewriteDataFiles checkAndApplyFilter(
       RewriteDataFiles action, String where, String tableName) {
     if (where != null) {
       try {
-        Expression expression =
-            SparkExpressionConverter.collectResolvedSparkExpression(spark(), tableName, where);
-        return action.filter(SparkExpressionConverter.convertToIcebergExpression(expression));
+        Option<Expression> expressionOption =
+                SparkExpressionConverter.collectResolvedSparkExpressionOption(spark(), tableName, where);
+        if (expressionOption.isEmpty()) return action.filter(Expressions.alwaysFalse());
```

Comment:
Nit: I'm not sure if checkstyle/spotless will fail here, but think we need extra newline in any case for the return.

Reply:
yes. you are right. the style check failed here.  i need to add `{}` for each `if` statement.